modular_skyrat/modules/customization/modules/clothing/suits/misc.dm

```diff
@@ -58,7 +57,7 @@
 	icon_state = "hawaiian_purple"
 	desc = "Strangely en vouge with aviator wearing shibas."
 	body_parts_covered = CHEST|GROIN
-	mutant_variants = NONE
```

Comment:
Why are you removing these?

Reply:
Oh, I thought another pr done that already and I wanted to copy over the changes, I'll re addit